beets/ui/commands.py

```diff
@@ -92,6 +92,15 @@ def _print_rows(names):
     print_("Album fields:")
     _print_rows(library.Album.all_keys())
 
+    def _print_sqlite(query):
+        [print_(' ' * 2 + dict(zip(row.keys(), row))['key']) for row in query]
```

Comment:
As a style matter, an imperative loop probably shouldn't be a written as a list comprehension—a normal for loop will likely be more readable.

Also, since this function doesn't close over any state from the outer scope, it should probably be a top-level function (rather than nested).